### ISSUE TITLE
Feature Request: Visual line mode

### ISSUE THREAD

**Issue by joshzcold** [Davidyz/inlayhint-filler.nvim#2]
Thank you for the plugin. I love it ♥️

Currently inlayhint-filler.nvim only works in normal visual mode, but it would be nice to include support for visual line mode.

Here https://github.com/Davidyz/inlayhint-filler.nvim/blob/2017f4fd441be67137793c78211991fdd501e715/lua/inlayhint-filler.lua#L87

If we could get the visual line representation of visual line 
```lua
    local start_pos = vim.fn.getpos("v")
    local end_pos = vim.fn.getpos(".")
    ---@type lsp.Range
    local lsp_range = {
      start = { line = start_pos[2] - 1, character = start_pos[3] - 1 },
      ["end"] = { line = end_pos[2] - 1, character = end_pos[3] - 1 },
    }
```
The I believe we can pass the new `range` into `vim.lsp.inlay_hint` and reuse the same logic (or do something similar)

**Comment by Davidyz** [comment]
Ok I will look into this. I usually use incremental selection provided by treesitters so I completely missed it. Thanks for pointing it out.

**Comment by Davidyz** [comment]
Hi, please try the new commit in `line-selection` branch and lmk if it works as intended.

**Comment by joshzcold** [comment]
@Davidyz working great. Thank you! 👍

**Comment by Davidyz** [comment]
Merged (#3). I'll delete the `line-selection` branch to keep things clean. Don't forget to switch back to `main`.
